tests/Http/Role/AddLabelToRoleRequestHandlerTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\Entity;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Label\ValueObjects\Privacy;
+use CultuurNet\UDB3\Label\ValueObjects\Visibility;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use CultuurNet\UDB3\StringLiteral;
+use Fig\Http\Message\StatusCodeInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+    use AssertJsonResponseTrait;
+
+    private AddLabelToRoleRequestHandler $handler;
+
+    private TraceableCommandBus $commandBus;
+
+    /**
+     * @var ReadRepositoryInterface | MockObject
+     */
+    private $labelRepository;
+
+    protected function setUp()
```

Comment:
This method can have a `void` return type. PHPStorm helps me to spot these things ☺️